study.md

```diff
@@ -44,21 +45,24 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+When developers talk about views, they are referring to the different pages which contain HTML fragments.
+Everything the user sees on a webpage is a view.
+Routers help the users navigate from point A to point B.
 ```
 
 ## Define the advantages of an SPA over a traditional application
 
 What are the advantages of a single page application over a traditional appliction?
 
 ```md
-<!-- your answer here -->
+SPAs are more responsive and free of reloading pages compared to traditional applications.
+SPAs also use JSON which separates the HTML from the app logic, making it easier to design each layer in the future.
 ```
 
 ## List some technologies used to make SPA's
 
 What are some of the tools used to create single page applications?
 
 ```md
-<!-- your answer here -->
+Is this referring to the frameworks such as Angular, Durandal and Ember?
```

Comment:
Those are definitely important, as well as HTML, CSS, JavaScript, AJAX, and JSON.